internal/command/container.go

```diff
@@ -59,6 +58,7 @@ type containerImage interface {
 	Tags() []string
 
 	Engine() containerEngine
+	Debugger() debugger
```

Comment:
This looks more like moved rather than removed. In this new position all containerImages must be debuggable. Is this a desirable definition of an image?
Go loose typing indicates that anything could be debuggable, if it provides the facilities to be debugged, rather than stating which interfaces will have to support debugging...

Reply:
Hum, I am not sure that I get what you want correctly. Do you mean, I should always cast all Debugger call on the Engine()? Something like: `debug, ok := image.Engine().(Debugger)` and then if the debugger is present do call the debugger function?